roles/openshift_health_checker/openshift_checks/logging/logging.py

```diff
@@ -64,6 +64,17 @@ def not_running_pods(pods):
         ]
 
     @staticmethod
+    def running_pods(pods):
+        """Returns: list of pods in a running state"""
+        return [
+            pod for pod in pods
+            if any(
+                condition['type'] == 'Ready' and condition['status'] == 'True'
+                for condition in pod['status']['conditions']
```

Comment:
I didn't recall the `conditions` field... is Ready==True equivalent to Phase==Running?

[vendor/k8s.io/kubernetes/pkg/api/v1/types.go#L2453-L2461](https://github.com/openshift/origin/blob/a192b3e406444e23ac12b91354fd0c0130e6e2ae/vendor/k8s.io/kubernetes/pkg/api/v1/types.go#L2453-L2461)